org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/RunnableDeviceServiceImpl.java

```diff
@@ -51,50 +51,50 @@
 
 @SuppressWarnings("rawtypes")
 public final class RunnableDeviceServiceImpl implements IRunnableDeviceService, IScanService {
-	
+
 	private static final Logger logger = LoggerFactory.getLogger(RunnableDeviceServiceImpl.class);
-	
+
 	/**
 	 * The default Malcolm Hostname can be injected by spring. Otherwise
 	 * this machine will be used for the malcolm host for instance 'beamline'-control.
 	 */
 	public static String defaultMalcolmHostname = null;
-	
+
 	/**
 	 * This service can not be present for some tests which run in OSGi
 	 * but mock the test laster.
 	 */
-	private static IScannableDeviceService deviceConnectorService;	
-	
+	private static IScannableDeviceService deviceConnectorService;
+
 	/**
 	 * This service must be present.
 	 */
 	private static IMalcolmService         malcolmService;
-	
+
 	/**
 	 * Map of device model class to device class.
 	 * NOTE This is not unmodifiable. Entries may be made after service create time. For instance
 	 * when the spring files are parsed.
 	 */
 	private static final Map<Class<?>, Class<? extends IRunnableDevice>> modelledDevices;
-	
+
 	/**
 	 * Map of device name to created device. Used to avoid
 	 * recreating non-virtual devices many times.
-	 * 
+	 *
 	 * TODO Should this be populated by spring?
 	 */
 	private static final Map<String, IRunnableDevice> namedDevices;
-	
-	
+
+
 	// This field is used to provide the getActiveScanner() method on the service.
 	// It should not be accessed from elsewhere.
 	private static IRunnableDevice<?> currentScanningDevice;
 
-	
+
 	// Use a factory pattern to register the types.
 	// This pattern can always be extended by extension points
-	// to allow point generators to be dynamically registered. 
+	// to allow point generators to be dynamically registered.
 	static {
 		System.out.println("Starting device service");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)